pyannote/audio/interactive/common/controller.js

```diff
@@ -28,14 +28,15 @@ var refresh = true;
 
 var left = 'ArrowLeft';
 var right = 'ArrowRight';
-var startR = 'Shift';
-var endR = 'Control';
+var startR = prodigy.config.left || 'Shift';
```

Comment:
Where does this `prodigy.config.left` come from?

Reply:
From the recipe.py : 
`
return {
    ...
    "config" : {
        ...
        "left" : left,
    }
}
`